src/backend/aspen/api/views/tests/test_update_phylo_run_and_tree.py

```diff
@@ -54,7 +61,7 @@ async def make_shared_test_data(
 
     await async_session.commit()
 
-    return user, group, samples, phylo_run, phylo_tree
+    return user, group, samples, phylo_run, phylo_tree, pathogen
```

Comment:
It looks like there are a lot of calls to this function scattered around our tests that need updating to handle this new signature.